### ISSUE TITLE
RCLootCouncil crashes when looting items (v0.10.1)

### ISSUE THREAD

**Issue by Nenuvah** [evil-morfar/RCLootCouncil_Classic#27]
When looting some white items RCLootCouncil randomly crashes

```
Date: 2020-06-04 10:19:43
ID: 2
Error occured in: Global
Count: 1
Message: ML.lootQueue nil
Debug:
   (tail call): ?
   [C]: error()
   ...tCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:45: ThrowError()
   ...tCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:40:
      ...tCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:39
   (tail call): ?
   ...\AddOns\RCLootCouncil_Classic\RCLootCouncil\core.lua:1926: ?()
   ...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:119:
      ...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:119
   [C]: ?
   ...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:29:
      ...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:25
   ...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:64: Fire()
   ...AddOns\Bartender4\libs\AceEvent-3.0\AceEvent-3.0.lua:120:
      ...AddOns\Bartender4\libs\AceEvent-3.0\AceEvent-3.0.lua:119
Locals:
None
AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AtlasLootClassic, vv1.5.1
  AtlasLootClassicData, vv1.5.1
  AtlasLootClassicDungeonsAndRaids, vv1.5.1
  AtlasLootClassicPvP, vv1.5.1
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucScanData, v8.2.6365 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  BagBrother, v
  Bartender4, v4.9.0
  BCT, v1.29
  BeanCounter, v8.2.6434 (SwimmingSeadragon)
  ClassicAuraDurations, v1.13.56
  ClassicCastbars, vv1.3.7
  ClassicCastbarsOptions, v
  Combuctor, v8.3.2
  DBMCore, v1.13.47
  DBMDefaultSkin, v
  DBMPartyClassic, v
  DBMStatusBarTimers, v
  Details, v
  Enchantrix, v8.2.6428 (SwimmingSeadragon)
  EnchantrixBarker, v8.2.6469 (SwimmingSeadragon)
  GatherMate2, v1.45.5
  Grid2, vr955
  Grid2LDB, vr955
  Grid2RaidDebuffs, vr955
  Informant, v8.2.6374 (SwimmingSeadragon)
  ItemRack, v
  NovaWorldBuffs, v1.60
  Questie, v5.10.3
  RCLootCouncilClassic, v0.10.1
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  Stubby, v8.2.6376 (SwimmingSeadragon)
  ThreatClassic2, v2.19
  TrinketMenu, v7.3.2
  Vendor, v3.5.8
  VendorPrice, v1.3.5
  WeakAuras, v2.17.5
  WeaponSwingTimer, v4.1.0
  BlizRuntimeLib_enUS v1.13.4.11304 <none>
  (ck=5bc)


```

**Comment by evil-morfar** [comment]
That was supposed to be silent and only show up in the logs.

It's pretty much harmless though, as I only added it to investigate why that particular variable isn't always initialized.

Thanks for letting me know.

**Comment by evil-morfar** [comment]
Fixed in v0.11.0
